index.rst

```diff
@@ -12,18 +12,633 @@
+
+
+.. figure:: /_static/figures/fake_snr_over_pred_snr_vs_fakemag_templt_and_science.png
+    :name: fig-fake-snr_over-pred-snr-vs-fakemag-templt-and-science
+    :target: ../_images/fake_snr_over_pred_snr_vs_fakemag_templt_and_science.png
+
+    The ratio of the `Model SNR` to the pipeline reported `SNR`, for each subtraction method and as a function of the pipeline reported `SNR` and the Fake Magnitude. We also split between science and template fakes.
+
+.. figure:: /_static/figures/fake_snr_over_est_snr_vs_fakemag_templt_and_science.png
+    :name: fig-fake-snr_over-est-snr-vs-fakemag-templt-and-science
+    :target: ../_images/fake_snr_over_est_snr_vs_fakemag_templt_and_science.png
+
+    The ratio of the variance `estimated SNR`` to the pipeline reported `SNR`, for each subtraction method and as a function of the pipeline reported `SNR` and the Fake Magnitude. We also split between science and template fakes.
+
+
+
+Efficiency estimation
+---------------------
+
+We calculate the efficiency of detecting the fake injected sources as a function of the Model and estimated SNR values.
+
+Our fitting procedure is using the binomial distribution model for the detection of a fake transient, this results in a sigmoid function model :math:`\rm{eff}(x) = [1 + e^{-a (x - b)}]`, and our error estimation is according to the variance of this distribution.
+
+.. figure:: /_static/figures/eff_estimation_diffim_flavors.png
+    :name: fig-eff-estimation-diffim-flavors
+    :target: ../_images/eff_estimation_diffim_flavors.png
+
+    The efficiency of the pipelines for finding fake transient sources, for the different DIA flavors. The SNR used is the `Model SNR`.
+
+.. figure:: /_static/figures/eff_estimation_diffim_flavors_estsnr.png
+    :name: fig-eff-estimation-diffim-flavors-estsnr
+    :target: ../_images/eff_estimation_diffim_flavors_estsnr.png
+
+    The efficiency of the pipelines for finding fake transient sources, for the different DIA flavors. The SNR used is the `estimated SNR`.
+
+We observe that for each of the SNR estimations the efficiency measurement lacks signal in the bins with low-SNR (from 0 to 5), where apparently we have not enough sources to obtain a real efficiency profile function.
+
+We define the SNR at 50% efficiency :math:`SNR_{1/2}`, and from our modelling we obtain that it is oscillating just below :math:`SNR_{1/2} = 5`. We might be observing in this case the SNR detection threshold that is configured by default in the pipelines.
+
+Our results point also that the efficiency is good at :math:`SNR>5`, but it is not reaching a stable value, and it oscilates dropping up to an 80%. This should instead be approaching to a 100% upper bound.
```

Comment:
It is probably worth pointing out that the number of fakes used here was too low for robust statistics, and that future work (well, now current) will increase the number of fakes.

Reply:
Done! Thanks